### ISSUE TITLE
Can't display dynamically in full screen

### ISSUE THREAD

**Issue by lukewlms** [inProgress-team/react-native-youtube#252]
I'd like to display a video in full screen when a button is pressed.

When the video is present on component mounting, it works fine - it just plays full screen immediately.  However, of course I need this to happen in response to user input.

Trying dynamically setting play and fullscreen is not working - the video never appears.

            <YouTube            
              videoId={videoId}
              play={this.state.isVideoShowing}
              fullscreen={this.state.isVideoShowing}
              loop={true}
              ref={(youTube: any) => (this.youTube = youTube)}
            />

I'm also trying to only mount the YouTube component at all when the user clicks, but this also does not work - though this seems like it should!

    { !this.state.isVideoShowing ? null :
            <YouTube            
              videoId={videoId}
              play={true}
              fullscreen={true}
              loop={true}
              ref={(youTube: any) => (this.youTube = youTube)}
            />
    }

Any guidance on why the video is never showing up unless I have it play when the parent component mounts?

**Comment by davidohayon669** [comment]
@lukewlms iOS?
from the README

> `reloadIframe()` (iOS): Specific props (fullscreen, modestbranding, showinfo, rel, controls, origin) can only be set at mounting and initial loading of the underlying WebView that holds the YouTube iFrame (Those are <iframe> parameters). If you want to change one of them during the lifecycle of the component, you should know the usability cost of loading the WebView again, and use this method right after the component received the updated prop.

TL;DR try calling `reloadFrame()` on the <YouTube /> instance **after** changing fullscreen prop

**Comment by lukewlms** [comment]
That looks helpful - thanks!
